src/main/java/movie/storage/model/CinemaHall.java

```diff
@@ -5,9 +5,7 @@
 import javax.persistence.GenerationType;
 import javax.persistence.Id;
 import javax.persistence.Table;
-import lombok.Data;
 
-@Data
```

Comment:
why have you decided to remove this for all the models?

Reply:
Added again